src/sass/global.scss

```
@@ -0,0 +1,35 @@
+body {
+  font-size: 10px;
+  font-family: "Roboto", sans-serif;
+  line-height: 1;
+}
+p {
+  font-family: Arial, Helvetica, sans-serif;
+}
+.container {
+  max-width: 1140px;
+  margin: 0 auto;
+}
+.btn {
+  text-transform: uppercase;
+  color: white;
+  border: none;
+  &__primary {
+    background-color: $yellow;
+  }
+  &__secondary {
+    background-color: $black;
+  }
+  &__outline {
```

```suggestion
.btn {
  text-transform: uppercase;
  color: white;
  border: none;
  
  &__primary {
    background-color: $yellow;
  }
  
  &__secondary {
    background-color: $black;
  }
  
  &__outline {
```
Mỗi block cách nhau 1 dòng nhé, tập dần thói quen luôn. Với lại cái đống style này cho vào file `/src/sass/components/_button.scss` nhé